src/domain/services/redis.listener.ts

```diff
@@ -37,6 +37,13 @@ export class RedisListener {
         [
           // XREAD can read from multiple streams, starting at a
           // different ID for each...
+          {
+            key: 'product_updates',
+            id: lastMessageId,
+          },
+          // Following can be deleted after PO is updated to use generic stream name
+          // Note should strictly have a different message id but PO will take more than one millisecond to switch
```

Comment:
We won't switch all 4 flavors at the exact same time, hopefully it won't be an issue as the non food products will be ignored anyway.

Reply:
This was more because PO will switch from using the product_updates_off stream to product_updates. In theory the message id is only unique per stream, but it is prefixed with the time in milliseconds so using the same "last" message id to track the transition shouldn't be a problem.